src/styles/components/header.scss

```diff
@@ -1,2 +1,95 @@
+@import '../mixins';
+
 .header {
+    background: var(--color-background-darker);
+    font-size: 1.6rem;
+    border-bottom: 0.1rem solid #5d5c61;
+    font-weight: var(--weight-semi-bold);
+    
+
+    &__content{
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-shrink: 5;
+        margin-left: 3rem;
```

Comment:
remove the margin and padding from `header__content-item` and move it to `header__content`

Reply:
Completed